src/server.js

```diff
@@ -239,12 +244,12 @@ function auth(data) {
 			if (success) {
 				var token;
```

Comment:
Forgot to remove this variable as I removed the need to pass around `token` into `init`.